packages/better-auth/src/plugins/organization/organization.ts

```diff
@@ -201,13 +210,13 @@ export const organization = <O extends OrganizationOptions>(options?: O) => {
 		setActiveOrganization,
 		getFullOrganization,
 		listOrganization,
-		createInvitation,
+		createInvitation: createInvitation(options as O),
 		cancelInvitation,
 		acceptInvitation,
 		getInvitation,
 		rejectInvitation,
 		removeMember,
-		updateMemberRole,
+		updateMemberRole: updateMemberRole(options as O),
```

Comment:
```suggestion
		updateMemberRole: updateMemberRole(options),
```
Link to : https://github.com/better-auth/better-auth/pull/565/files#r1845484912